lib/iris/plot.py

```diff
@@ -1892,6 +1952,11 @@ def animate(cube_iterator, plot_func, fig=None, **kwargs):
     >>> ani = iplt.animate(cube_iter, qplt.contourf)
     >>> iplt.show()
 
+    Notes
+    ------
+    This function does not maintain laziness when called; it realises data.
+    See more at :doc:`/userguide/real_and_lazy_data`.
+
     """
```

Comment:
@ESadek-MO There are no statements about laziness for the following `iris.plot` public API functions:

- [ ] `orography_at_bounds`
- [ ] `orography_at_points`
- [ ] `fill_between`
- [ ] `hist`
- [ ] `symbols`
- [ ] `animate`

Is this because they do maintain laziness?

Reply:
@ESadek-MO Just need to consider `animate` and we're done here :+1: